examples/examples-cpu/nyc-taxi-snowflake/xgboost-dask.ipynb

```diff
@@ -342,10 +340,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from sklearn.metrics import mean_squared_error\n",
-    "\n",
     "preds = xgb_reg.predict(taxi_test[features])\n",
-    "mean_squared_error(taxi_test[y_col].compute(), preds.compute(), squared=False)"
+    "mse = mean_squared_error(taxi_test[y_col].to_dask_array(), preds, squared=False)\n",
+    "print(f\"MSE: {round(mse, 4)}\")"
```

Comment:
I think it's clearer without the print statement since we are already in a notebook.

Reply:
ok